---
output: 
  bookdown::pdf_document2:  
    number_sections: false
  bookdown::word_document2:  
    number_sections: false
toc: false
params:
  lastmod: !r lubridate::now()  
---


# SPECIFIC AIMS

Mouse multiparental populations have revolutionized systems genetics by incorporating
diverse genetic variants into mapping cohorts, enabling deep phenotyping, and enhancing 
mapping resolution and biological reproducibility. 
In particular, multiparental populations are used extensively to study infectious diseases. While genotype and 
phenotype data are available for a variety of infections, analytic methods that enhance statistical 
power to detect genetic loci (quantitative trait loci, QTL) that affect these infection-related traits are still lacking.

Two complementary mapping populations, the
Collaborative Cross and the Diversity Outbred mice, arise from the same eight inbred 
founder lines and, therefore, share the same collection of genetic variants. However, the two mapping cohorts 
differ in genetic architecture and possess complementary strengths. The Collaborative Cross is a collection of 
recombinant inbred lines that allow experimental replication and serial phenotyping of 
genetically identical subjects, while the 
Diversity Outbred is a collection of highly heterozygous and genetically unique 
mice with small recombination intervals.

Both Collaborative Cross and Diversity Outbred populations have been used to model tuberculosis pathogenesis.  
Our colleagues have performed deep 
phenotyping of Collaborative Cross mice after infection with Mycobacterium tuberculosis (Mtb), and our collaborators have quantified Mtb 
susceptibility in a large panel of Diversity Outbred animals. In both cases, Mtb infection produces a wide range of 
outcomes, which in some cases can be associated with QTL.  However, the lack of concordance between these studies 
suggests that many more QTL might be observed with more sensitive methods.

We propose an analytic framework that leverages strengths of both Collaborative Cross and Diversity Outbred mice. 
By first developing a strategy to infer allelic series at a QTL in Diversity Outbred mice, we will then be able to probe the Collaborative Cross with enhanced statistical power. This strategy will increase the resolution of mapping in the Collaborative Cross panel
and allow susceptibility-related traits from the Diversity Outbred to be associated with more specific phenoytpes in the Collaborative Cross.  






**Specific Aim 1.** *METHODS: Design and implement new biostatistical methods for parametric bayesian inference of allelic series at established QTL in multiparental populations.* 

Methods to quantify distinct QTL alleles are needed to enhance statistical power to detect QTL. To address this need, 
we propose a novel Bayesian statistical framework for inferring allelic series at QTL. Knowing the allelic series, 
*i.e.*, allelic number and configuration, enables QTL mapping with fewer statistical parameters, and, thus, enhanced 
statistical power.


**Specific Aim 2.** *APPLICATION: Apply allelic series inferences to complex traits from Diversity Outbred and Collaborative Cross mice.* 

We first will perform simulation studies to characterize QTL detection power 
enhancements with allelic series-informed QTL mapping. We then will apply our 
methods to tuberculosis-related traits in Diversity Outbred and Collaborative Cross 
mice. We will leverage initial results in each mapping cohort to inform mapping methods in the other.

**Specific Aim 3.** *SOFTWARE: Create and share an open-source software package that implements our methods.* 

We will create and share a user-friendly, open-source R software package and share it via Github. 
Additionally,
we will create a website that hosts and documents our software.
To encourage durability of our software and methods, we'll use Docker and Dockerhub
to execute and share our analyses and to test our code. 
